decorators/decorators-test-helpers.js

```diff
@@ -26,6 +26,21 @@ function testDecoratorGetter(decoratorName, decorator, propName, getter, tester)
 			tester(TesterType);
 			QUnit.equal(ran, true, "getter ran");
 		});
+
+		QUnit.test(decoratorName + " getter decorator with class Object (without decorator)", function() {
+			var ran = false;
+
+			class TesterType extends ObserveObject {
```

Comment:
Is there something special we do to make `class` syntax work alongside the tests in the rest of CanJS that are expected to run in IE 11?

Reply:
IE 11 doesn't support the class syntax, so I'm not sure what it would do. I've opened #78 to that effect though.